### DISCUSSION TITLE
Templates download detected as virus.

### DISCUSSION THREAD

**Discussion by mcurros2** [projectdiscovery/nuclei-templates#1673] [General]
Hi,

CVE-2017-12615.yaml is detected as virus, PHP backdoor shell. Any suggestions?

Regards!

**Comment by geeknik** [comment]
Well, obviously the template isn't a virus, so you can always [add an exclusion to Windows Security](https://support.microsoft.com/en-us/windows/add-an-exclusion-to-windows-security-811816c0-4dfd-af4a-47e4-c301afe13b26).

**Reply by geeknik** [reply]
Additionally, you can submit the template file as a false positive to Microsoft [here](https://www.microsoft.com/en-us/wdsi/filesubmission). 👍🏻
